public/wallet/index.html

```diff
@@ -1,86 +1 @@
-<!DOCTYPE html>
```

Comment:
Would be easier to review if not minified. I doubt it changes the compressed size much either.